src/game.cpp

```diff
@@ -1871,7 +1876,10 @@ void Game::processKeyInput()
 	} else if (wasKeyDown(KeyType::CMD)) {
 		openConsole(0.2, L"/");
 	} else if (wasKeyDown(KeyType::CMD_LOCAL)) {
-		openConsole(0.2, L".");
+		if (client->moddingEnabled())
+			openConsole(0.2, L".");
+		else
+			m_game_ui->showStatusText(wgettext("CSM Disabled"));
```

Comment:
possibly `CSM is disabled`, unlike other hotkeys, one can not simply switch it by pressing it.